### PR TITLE
Lose spilo weight and add postgres 10

### PR DESCRIPTION
**Pull request by CyberDem0n**
+ add some useful extensions and contribs

### CODE REVIEW

**Comment by CyberDem0n** [comment]
I did it for purpose, because we don't want to rebuild them every time.
If somebody don't trust - it's always possible to remove deb files and trigger rebuild.

File: postgres-appliance/Dockerfile.build [hunk 1]
```diff
@@ -0,0 +1,421 @@
...
+
+    && git clone -b $PLPROFILER https://bitbucket.org/openscg/plprofiler.git \
+
+    && git clone -b $TIMESCALEDB https://github.com/timescale/timescaledb.git \
+
+    && git clone -b $PGEXTWLIST https://github.com/dimitri/pgextwlist.git \
+    && perl -ne 'print unless /PG_TRY/ .. /PG_CATCH/' pgextwlist/pgextwlist.c > pgextwlist.c.f \
+    && egrep -v '(PG_END_TRY|EmitWarningsOnPlaceholders)' pgextwlist.c.f > pgextwlist/pgextwlist.c \
+
+    && git clone https://github.com/pgq/skytools-legacy.git \
```
Comment (alexeyklyukin) [inline postgres-appliance/Dockerfile.build]: you can do it with git clone --recurse-submodules

File: postgres-appliance/build.sh [hunk 2]
```diff
@@ -24,11 +38,34 @@ cat > scm-source.json <<__EOT__
...
+            | awk '{print $2; exit 0}')
+
+    # new "-build" image has the same id as already exiting one
+    [[ ! -z $TAG_OF ]] && docker tag ${TAG_OF%-build}-squashed $IMGNAME-squashed && return 0
+
+    [[ "$OLD_BUILD_ID" != "$NEW_BUILD_ID" || -z "$(docker images -q $IMGNAME-squashed)" ]] \
+            && run_or_fail docker-squash -t $IMGNAME-squashed $IMGNAME-build
+}
+
+run_or_fail docker build "${build_args[@]}" -f Dockerfile.build
```
Comment (alexeyklyukin) [inline postgres-appliance/build.sh]: I think this is missing the trailing '.' as the directory where the Docker should operate (also on line 69). At least it didn't work on my system without it.

Comment (CyberDem0n) [reply postgres-appliance/build.sh]: This build script accepts the same arguments as "docker build".
Basically it is a replacement for "docker build".
I invoke it as: `./build.sh --build-arg COMPRESS=true -t my-cool-image:my-cool-tag .`

File: postgres-appliance/Dockerfile.build (hunk 1 above)
Comment (CyberDem0n) [reply postgres-appliance/Dockerfile.build]: Fixed. Thank you for the tip.

**Comment by alexeyklyukin** [comment]
👍

**Comment by CyberDem0n** [comment]
:+1:

**Pull request merged**